README.rst

```diff
@@ -1,6 +1,43 @@
 Open edX Brand Package Interface
 ================================
 
-This project contains the default branding and style used in Open edX applications.
+This project contains the default branding assets and style used in Open edX applications. It is published on npm as `@edx/brand-openedx`.
 
 The file structure serves as an interface to be implemented for custom branding and theming of Open edX.
+
+This project contains branding assets and themes for edx.org.
```

Comment:
Open edX?

Reply:
Thanks I removed the whole sentence since it was duplicative.